backend/src/main/java/gov/cdc/usds/simplereport/config/SecurityConfiguration.java

```diff
@@ -96,6 +96,11 @@ public IdentitySupplier getRealIdentity() {
         }
         LOG.debug("Hello JWT user {} {} ({})", firstName, lastName, email);
         return new IdentityAttributes(email, firstName, null, lastName, null);
+      } else if (principal instanceof String) {
+        String principalString = (String) principal;
+        if ("anonymousUser".equals(principalString)) {
```

Comment:
Is this a constant somewhere in Spring Security?

Reply:
It doesn't appear to be, they use the raw string all over their own source code. 🤦 

I'm double-checking though

Reply:
Indeed, no – it's just the raw string and they do not expose a constant for our use